config/platforms.yaml

```diff
@@ -331,6 +331,20 @@ platforms:
     dtb: dtbs/qcom/qcs6490-rb3gen2.dtb
     compatible: ['qcom,qcs6490-rb3gen2', 'qcom,qcm6490']
 
+qcs8300-ride:
```

Comment:
You need to indent this new device/platform name a little bit.

Have a look at where some of the before and after start in terms of indentation and whitespace.

YAML is unfortunately white spaces sensitive.